src/applications/personalization/account/components/AccountMain.jsx

```diff
@@ -25,7 +25,7 @@ class AccountMain extends React.Component {
   }
 
   renderMVIError() {
-    if (this.props.profile.veteranStatus === 'OK') {
+    if (this.props.profile.status === 'OK') {
       return null;
```

Comment:
Hm, any idea what the distinction is?

Reply:
veteran status is whether the user was found in  EMIS